src/test/java/uk/gov/companieshouse/filinghistory/api/controller/AnnotationTransactionIT.java

```diff
@@ -276,7 +299,7 @@ void shouldInsertNoParentAnnotationDocumentAndReturn200OK() throws Exception {
 
         FilingHistoryDocument actualDocument = mongoTemplate.findById(TRANSACTION_ID, FilingHistoryDocument.class);
         assertNotNull(actualDocument);
-        expectedDocument.updatedAt(actualDocument.getUpdatedAt());
+        expectedDocument.updated(actualDocument.getUpdated());
```

Comment:
301 and 302 can be removed, as we are stubbing the call to the instant supplier.

Reply:
all instances removed.